config/config.js

```diff
@@ -88,6 +88,28 @@ var validateSecureMode = function (config) {
 };
 
 /**
+ * Validate Session Secret parameter is not set to default in production
+ */
+var validateSessionSecret = function (config, testing) {
```

Comment:
@ilanbiala you're missing the point. I'm not saying it's better or worse, I'm saying that the entire config.js file (https://github.com/jloveland/mean/blob/local-session-secret/config/config.js) makes use of the same style functions, i.e:
```javascript
var getGlobbedPaths = function (globPatterns, excludes) {}
var validateEnvironmentVariable = function () {}
var validateSecureMode = function (config) {}
var validateSessionSecret = function (config, testing) {}
var initGlobalConfigFolders = function (config, assets) {}
and so on
```

I don't think that this PR or others should turn into code style PRs. If we want to change this we can opt for another PR to update to better code styles.